src/Uri.php

```diff
@@ -265,6 +265,12 @@ public static function isSameDocumentReference(UriInterface $uri, UriInterface $
      */
     public static function removeDotSegments($path)
     {
+        @trigger_error(
+            'GuzzleHttp\Psr7\Uri::removeDotSegments is deprecated since version 1.4. ' .
```

Comment:
I mentioned because you referred to Symfony deprecation standard which includes this.